Add warning for hyphenated project names (#67)
stellar/soroban-docs: docs/getting-started/quick-start.mdx

A `.wasm` file should be outputted in the `target` directory:

```
target/wasm32-unknown-unknown/release/first_project.wasm
```

> :::caution

Reviewer: Let's make this an `info` block.

Reviewer: ```suggestion
:::info
```

Reviewer: ```suggestion
:::info
```

> Hyphens get replaced by underscores, so `first-project` is outputted as `first_project.wasm`.

Reviewer: ```suggestion
Hyphens in Rust crate names are replaced by underscores in code and generated file names, so `first-project` is outputted as `first_project.wasm`.
```
